packages/@tinacms/toolkit/src/packages/fields/plugins/ImageFieldPlugin.tsx

```diff
@@ -97,14 +97,13 @@ export const ImageField = wrapFieldsWithMeta<InputProps, ImageProps>(
         props.input.onChange(parsedValue)
       }
     }
-
     const uploadDir = props.field.uploadDir || (() => '')
 
     return (
       <ImageUpload
         value={value}
         previewSrc={src}
-        loading={srcIsLoading}
```

Comment:
`srcIsLoading` doesn't appear to be evaluating to true during an image upload / update and so we're not getting the loading dots as any kind of visual affordance in the side bar.

Maybe it's not working as expected or my understanding of what it's doing is off. @dwalkr any insights on this one?

Reply:
Pushed a few changes that remove the extra `imgLoading` check in favor of the `srcIsLoading`. Able to consistently drag and drop files to upload and set as the image and see it reflected.

The boolean value isn't dynamically changing, and as such there is not visual affordance to indicate we're waiting on a file upload (it does take a few seconds).

In short, it works. But need to add that extra bit of polish and feedback for the user.

Reply:
I agree w/ Jeff here, we likely need to keep track of two loading states:

1. retrieving the image URL asynchronously (use case: GithubClient + private repo), done by `usePreviewSrc`
2. uploading an image and waiting for it to be available on the server

we never really solved for 2 because I guess we weren't drag+droppers and were focused on using controls in the media manager interface